docker/demo/README.md

```diff
@@ -0,0 +1,55 @@
+# Lumify Demo Docker Container
+The Lumify Demo Docker container encapsulates a minimal deployment of Lumify.  The container is self contained and can be run independant of the development environment, after it is built.  
+
+## Prerequisites to build from source
+The following dependencies must be installed before building the Lumify Demo Docker Container.
+
+- OSX or Linux required to build
+- [Java 7 JDK](http://www.oracle.com/technetwork/java/javase/downloads)
+- [Maven](https://maven.apache.org/)
+- [Git client](http://git-scm.com/)
+- OSX build environments
+	- [VirtualBox](https://www.virtualbox.org/)
+	- [Boot2Docker (includes Docker)](http://boot2docker.io/)
+- Linux build environments
+	- [Docker](https://www.docker.com/)
+- HTML5 compliant web browser
+
+## Installation
+Execute the following commands to build the Lumify Demo Docker Container.
+
+```sh
+$ boot2docker init -m 8192
+$ boot2docker start 
+$ cd ~
```

Comment:
Need to indicate that boot2docker is only required for OS X, not Linux and that `boot2docker init` only needs to be run once. We should also include the shell variable export command, or is that not necessary for the demo VM?